### ISSUE TITLE
`"default": true` in `.markdownlint.jsonc` not respected

### ISSUE THREAD

**Issue by simensol** [DavidAnson/vscode-markdownlint#281]
According to [the documentation](https://github.com/DavidAnson/vscode-markdownlint#markdownlintconfig):

> The default rule configuration disables MD013/line-length because many files include lines longer than the conventional 80 character limit

However, if I create the following `.markdownlint.jsonc` file:

```json
{
    "default": true
}
```

markdownlint still complains about `MD013/line-length`. If I change the `.markdownlint.jsonc` file to:

```json
{
    "MD013": false
}
```

the linting error disappears. It seems that the `"default": true` in `.markdownlint.jsonc` is not respected?

**Comment by DavidAnson** [comment]
You're mixing two things here. :) In the library, all rules default to enabled and that property can be used to default them all to disabled. It works for that purpose as you demonstrate. In the VS Code plug-in, the MD013 rule is disabled by default to avoid being distracting in documents that have many long lines. However, if you manually configure all rules to be enabled as you are doing, then MD013 is enabled as well and it will report issues as you see.

**Comment by simensol** [comment]
It makes perfectly sense! Thanks for explaining :)

**Comment by simensol** [comment]
If `MD013` is disabled by default, an empty `.markdownlint.jsonc` shouldn't report any `MD013` issues, right? When I have this `.markdownlint.jsonc`:

```json
{
}
```

markdownlint raises `MD013` and I have to set

```json
{
    "MD013": false
}
```

to make the linting errors disappear.

**Comment by simensol** [comment]
Thanks for explaining—again it makes perfectly sense :) I guess I got confused by the "default rule configuration" sentence, and I didn't grasp (until now) that an empty `.markdownlint.json` actually replaces any "default rule configuration".
